### DISCUSSION TITLE
[Vue warn]: injection "Symbol(VueCesium)" not found.

### DISCUSSION THREAD

**Discussion by houluy** [zouyaoji/vue-cesium#250] [Q&A] [answered]
作者大大，我在一个非vc-viewer子组件中，仅仅调用一下useVueCesium，就会报标题所述warning，不过warning没有任何实质影响。请作者大大指点一下是否是我使用方式出错了？我使用的最新的3.0.16版本

```typescript
//TestPanel.vue
import { useVueCesium } from 'vue-cesium'
import type { VcViewerProvider } from 'vue-cesium/es/utils/types'
const $vc: VcViewerProvider = useVueCesium()
```

```html
<vc-viewer></vc-viewer>
<TestPanel v-if="xxx" @closePanel="xxx"></TestPanel>
```
Warning截图：
<img width="1433" alt="warning" src="https://user-images.githubusercontent.com/11565649/184275009-3b1ba07d-9c7c-4084-823a-5a7709228682.png">

这个TestPanel组件是在<vc-viewer>后面的一个组件。
谢作者大大！

**Comment by zouyaoji** [comment, marked as answer]
可能是改出bug了，我确认一下，如果是就下个版本修复一下。

**Reply by houluy** [reply]
辛苦啦～栓Q～

**Reply by zouyaoji** [reply]
已在v3.0.17修复。

https://zouyaoji.top/vue-cesium/#/zh-CN/component/quickstart#ru-he-zai-qi-ta-vue-zu-jian-zhong-huo-qu-viewer-shi-li

**Reply by houluy** [reply]
感谢~
